src/__tests__/components/CustomButton.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from '../../test-utils';
+import React from 'react';
+import CustomButton from '../../components/CustomButton/CustomButton';
+
+describe('Custom Button Tests', () => {
+  it('Should render the component', () => {
+    render(<CustomButton>Test</CustomButton>);
+    screen.getByText('Test');
```

Comment:
Jak taki test sprawdza czy prawidłowo wyrenderował się przycisk? Nie trzeba podglądać czy screen.getByText coś zwróci?

Reply:
Nie trzeba, jak nie znajdzie tego tekstu to wówczas wywali w teście error

Reply:
Ok dzięki za wytłumaczenie